src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -615,19 +627,22 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
     ) {
         val settings = pluginSettings()
 
-        val newOssTreeNodeText = when {
-            currentOssError != null -> "$OSS_ROOT_TEXT (error)"
-            isOssRunning(project) && settings.ossScanEnable -> "$OSS_ROOT_TEXT (scanning...)"
-            else -> ossResultsCount?.let { count ->
-                OSS_ROOT_TEXT + when {
-                    count == NODE_INITIAL_STATE -> ""
-                    count == 0 -> NO_ISSUES_FOUND_TEXT
-                    count > 0 -> " - $count vulnerabilit${if (count > 1) "ies" else "y"}$addHMLPostfix"
-                    else -> throw IllegalStateException()
+        val newOssTreeNodeText =
+            when {
+                ossResultsCount == null -> OSS_ROOT_TEXT
+                currentOssError != null -> "$OSS_ROOT_TEXT (error)"
+                isOssRunning(project) && settings.ossScanEnable -> "$OSS_ROOT_TEXT (scanning...)"
+
+                else -> ossResultsCount.let { count ->
+                    OSS_ROOT_TEXT + when {
+                        count == NODE_INITIAL_STATE -> ""
+                        count == 0 -> NO_ISSUES_FOUND_TEXT
+                        count > 0 -> " - $count vulnerabilit${if (count > 1) "ies" else "y"}$addHMLPostfix"
+                        else -> throw IllegalStateException()
+                    }
                 }
-            }
         }
-        newOssTreeNodeText?.let { rootOssTreeNode.userObject = it }
+        newOssTreeNodeText.let { rootOssTreeNode.userObject = it }
```

Comment:
Code above will not show `(scanning...)` and `(error)` states.
Also logic become inconsistent with other nodes and not following the method KDocs. I.e. `null` value for `ossResultsCount` should left node text as-is, due to it changes in other places, like `VulnerabilityTreeCellRenderer` 
Would suggest instead of that breaking changes do call 
`updateTreeRootNodesPresentation(ossResultsCount = NODE_INITIAL_STATE)` from scanningOssError()